src/oneD/StFlow.cpp

```diff
@@ -94,9 +94,12 @@ StFlow::StFlow(ThermoPhase* ph, size_t nsp, size_t points) :
     setupGrid(m_points, gr.data());
 
     // Find indices for radiating species
-    m_kRadiating.resize(2, npos);
+    m_kRadiating.resize(5, npos);
```

Comment:
There has been some work to make the identification of species more flexible (which would mean that lowercase species names would be recognized). Rather than hard coding names, specifying the elemental composition will in most cases work (there usually is only one isomer). The function name is [`findIsomers`](https://cantera.org/documentation/dev/doxygen/html/d6/d0e/classCantera_1_1Phase.html#a4dc30c10413ad6789dc45ac326c447c1)

Reply:
I changed this part.

It would be great if the user could select the components involved in the radiation himself.

It would probably be better to load the absorption coefficient data from a separate file, so that the user can add his own dependencies. 

Unfortunately, my experience with С++ is not enough.

Reply:
I agree that it would be great to select species / import parameters instead of hard coding them. There are likely multiple ways of getting this done. I can probably give you pointers, but it’s up to @bryanwweber and @speth ...